### PR TITLE
Hotfix: add type for recover, and some cleanups

### PR DESCRIPTION
**Pull request by jsonsivar**
Added the `type` field to the recover method and also updated the enum to be Google, Twitter, etc instead of Eth.

### CODE REVIEW

File: grpc/proto/space.proto [hunk 1]
```diff
@@ -584,6 +584,7 @@ message BackupKeysByPassphraseResponse {}
 message RecoverKeysByPassphraseRequest {
   string uuid = 1;
   string passphrase = 2;
+  KeyBackupType type = 3;
```
Comment (perfectmak) [inline grpc/proto/space.proto:587]: You should also update the `enum KeyBackupType` definition of the proto file to contain the new fields

**Review by perfectmak — CHANGES_REQUESTED** [review]
Generally looks good. I requested a change that you missed in the proto defintions.

**Review by maurycy — APPROVED** [review]
LGTM. 🚀

File: grpc/proto/space.proto (hunk 1 above)
Comment (jsonsivar) [reply grpc/proto/space.proto:589]: Good catch, fixed in [8f5fafc](https://github.com/FleekHQ/space-daemon/pull/260/commits/8f5fafc86fe33d2152906df537be2b92ef54bf80)

**Review by perfectmak — APPROVED** [review]
LGTM. There are some conflicts though.

**Pull request merged**